genetlink/Cargo.toml

```diff
@@ -0,0 +1,39 @@
+[package]
+name = "genetlink"
+version = "0.1.0"
+authors = ["Leo <leo881003@gmail.com>"]
+edition = "2018"
+homepage = "https://github.com/little-dude/netlink"
+repository = "https://github.com/little-dude/netlink"
+keywords = ["netlink", "linux"]
+license = "MIT"
+readme = "../README.md"
+description = "communicate with generic netlink"
+
+[features]
+default = ["tokio_socket"]
+tokio_socket = ["netlink-proto/tokio_socket","netlink-proto/workaround-audit-bug", "tokio"]
+smol_socket = ["netlink-proto/smol_socket","netlink-proto/workaround-audit-bug","async-std"]
+
+[dependencies]
+futures = "0.3.16"
+netlink-packet-generic = "0.1.0"
+netlink-proto = { default-features = false, version = "0.7.0" }
+tokio = { version = "1.9.0", features = ["rt"], optional = true }
+async-std = { version = "1.9.0", optional = true }
+bimap = "0.6.1"
```

Comment:
This is new dependency which can be easily replaced by two `HashMap`.
New dependency means more maintenance work for developer and package builder.
Please replace it with HashMap/etc

Reply:
The usage of bimap is removed.